packages/gateway/src/environment.ts

```diff
@@ -29,6 +29,7 @@ export const env = cleanEnv(process.env, {
   APPLICATION_CONFIG_URL: url({ devDefault: 'http://localhost:2021/' }),
   NOTIFICATION_URL: url({ devDefault: 'http://localhost:2020/' }),
   WORKFLOW_URL: url({ devDefault: 'http://localhost:5050/' }),
+  EVENTS_URL: url({ devDefault: 'http://localhost:5555/' }),
```

Comment:
Opinion: Since we are using both conventions, with and without trailing slash, I would opt for without when introducing new ones. What do you think?

Reply:
I agree; an extra trailing slash in our current setup might convey to a newcomer that it has a meaning that it doesn't